src/deps.cljs

```diff
@@ -1,4 +1,4 @@
 {:npm-deps
  {"complex.js" "^2.1.1"
-  "fraction.js" "^4.2.0"
+  "fraction.js" "^4.3.6"
```

Comment:
That's a good comment, and the developers of Fraction.js do suggest just "vendoring" the bigfraction.js if that's the API you want. I'm not sure quite where to put it.

On the other hand, the patches get swept in by the post install hook in package.json, so this feels like a principled approach...but if it does frustrate consumers of Emmy then it has to go.

Reply:
I experimented using a local "vendored" version of bigfraction.js: doing that is easy enough but it needs a package.json to go with it. Fine, but that seems morally equivalent to the patch-package technique I've used. Since CLJS compilation can't happen without `nom i`, and that will take care of applying the patch, are you still worried?

Reply:
My concern is about downstream libraries like Emmy-viewers - will all dependents now have to configure this patch as well? I would be fine pinning to an older version of Fraction as well to avoid this!